setup.ps1

```diff
@@ -28,15 +29,22 @@ Disable-ScheduledTasks
 Disable-IPv6To4
 if ($manual_install) {
     Disable-InternetExplorerESC
-    Edit-VisualEffectsRegistry
 }
+Edit-VisualEffectsRegistry
 Add-DisconnectShortcut
 
-Install-Chocolatey
-Install-VPN
-Join-Network $network
-Install-NSSM
+if ($network) {
+    Install-Chocolatey
```

Comment:
We can consider using chocolatey to install steam.

Reply:
I'm not overly familiar with chocolatey, does it bring many benefits over the current approach ? I though it was nice to be able to get rid of some unused software, but if you think it'd be better then we could do that indeed

Reply:
No really practical benefits. Implementation wise it's nice to centralize all software downloads to chocolatey. I'm fine with either.

Reply:
I think ZeroTier VPN is unneeded for most cases now, which means most deployments will need a single software download - Steam. Considering that, I do prefer the current manual approach as the Steam installer is very lightweight.

If we were to add more softs in the default install, it would be wiser to switch to choco for Steam indeed. I'll resolve this conversation, but input is still welcome if you decide you'd prefer to use chocolatey for steam too !